### PR TITLE
user register

### PR DESCRIPTION
**Pull request by nguyenhonghai1997bg**

### CODE REVIEW

File: app/Http/Controllers/Admin/ActiveUserController.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Repositories\UserRepository;
+
+class ActiveUserController extends Controller
+{
+    protected $user;
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/ActiveUserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.FunctionDeclaration.Found</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/ActiveUserController.php</br>**Line**: 13</br>Expected "function abc(...)"; found "function abc (...)"

Comment (framgiaciv3) [inline app/Http/Controllers/Admin/ActiveUserController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.SpaceBeforeOpenParen</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/ActiveUserController.php</br>**Line**: 13</br>Expected 0 spaces before opening parenthesis; 1 found

File: app/Http/Requests/UserFormRequest.php [hunk 2]
```diff
@@ -27,6 +27,7 @@ public function rules()
             'name' => 'required|min:3|max:255',
             'email' => 'required|unique:users,email,' . $this->id,
             'password' => 'required|string|min:6',
+            'password_confirm' => 'required|string|min:6|same:password'
```
Comment (framgiaciv3) [inline app/Http/Requests/UserFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/UserFormRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma

File: app/Repositories/RoleRepository.php [hunk 3]
```diff
@@ -10,4 +10,14 @@ public function model()
...
     }
+
+    public function pluckRole()
+    {
+        return $this->model->pluck('display_name', 'id');
+    }
+
+    public function getIdTrainee()
+    {
+    	return $this->model->select('id')->where('name', 'like', 'trainee')->first();
```
Comment (framgiaciv3) [inline app/Repositories/RoleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Repositories/RoleRepository.php</br>**Line**: 21</br>Spaces must be used to indent lines; tabs are not allowed

File: app/Models/Role.php [hunk 4]
```diff
@@ -41,4 +41,9 @@ public function permissions()
     {
         return $this->belongsToMany('App\Models\Permission', 'permission_role', 'role_id', 'permission_id');
     }
+
+    public function scopeGetIdRoleTrainee ($query)
```
Comment (framgiaciv3) [inline app/Models/Role.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.FunctionDeclaration.Found</br>**Severity**: error</br>**File**: app/Models/Role.php</br>**Line**: 45</br>Expected "function abc(...)"; found "function abc (...)"

Comment (framgiaciv3) [inline app/Models/Role.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.SpaceBeforeOpenParen</br>**Severity**: error</br>**File**: app/Models/Role.php</br>**Line**: 45</br>Expected 0 spaces before opening parenthesis; 1 found

**Comment by nguyenhonghai1997bg** [comment]
https://edu-redmine.framgia.vn/issues/5088

File: app/Repositories/RoleRepository.php (hunk 3 above)
Comment (framgiaciv3) [inline app/Repositories/RoleRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Repositories/RoleRepository.php</br>**Line**: 21</br>Spaces must be used to indent lines; tabs are not allowed

File: app/Http/Controllers/Admin/ActiveUserController.php [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        $this->user = $user;
+    }
+
+    public function activeUser(Request $request, $id)
+    {
+        $user = $this->user->find($id);
+        $user->status = config('site.active');
+        $user->save();
+
+        return response()->json(['message' => trans('Avtive User successfully')]);
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/ActiveUserController.php]: sai chính tả
Active

Comment (nguyenhonghai1997bg) [reply app/Http/Controllers/Admin/ActiveUserController.php]: =))

**Pull request merged**